code/network/multimsgs.cpp

```diff
@@ -77,6 +77,9 @@
 
 // #define _MULTI_SUPER_WACKY_COMPRESSION
 
+// where we might accidentally dereference nullptrs, send 0 instead
+constexpr float ZERO_VALUE = 0.0f;
```

Comment:
I know it's used in more than one function, but I'm not sure that's worth making it a global.

If it is a global though it needs to be `static`. And you might rename it to `ZERO_VALUE_float`, or something, to indicate it's type in case someone else needs an `int` version.

Reply:
I'll just move it back into the functions.